code/BlenderLoader.cpp

```diff
@@ -1118,12 +1118,13 @@ void BlenderImporter::ConvertMesh(const Scene& /*in*/, const Object* /*obj*/, co
             const aiFace& f = out->mFaces[out->mNumFaces++];
 
             aiColor4D* vo = &out->mColors[0][out->mNumVertices];
+			const float scaleZeroToOne = 1.f / 255.f;
             for (unsigned int j = 0; j < f.mNumIndices; ++j,++vo,++out->mNumVertices) {
                 const MLoopCol& col = mesh->mloopcol[v.loopstart + j];
-                vo->r = col.r;
-                vo->g = col.g;
-                vo->b = col.b;
-                vo->a = col.a;
+                vo->r = float(col.r) * scaleZeroToOne;
+                vo->g = float(col.g) * scaleZeroToOne;
+                vo->b = float(col.b) * scaleZeroToOne;
+                vo->a = float(col.a) * scaleZeroToOne;
             }
```

Comment:
Hi, could you please use `ai_real` instead of `float` here?
(`aiColor4D` is a `typedef` dependent on `ai_real` and should contain `double` when compiling for higher precision)